Examples/Swift/InsetMapExample.swift

```diff
@@ -0,0 +1,57 @@
+import Mapbox
+
+@objc(InsetMapExample_Swift)
+
+class InsetMapExample_Swift: UIViewController, MGLMapViewDelegate {
+    var mapView = MGLMapView()
+    var miniMapview = MGLMapView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        
+        // Set the main map's center coordinate and zoom level.
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 59.31, longitude: 18.06), zoomLevel: 9, animated: false)
+
+        // set inset mapview properties
```

Comment:
We should explain why we're doing this - what behavior does this result in? Its totally fine for examples to be verbose in comments.